crates/ruff/src/rules/flake8_bandit/rules/bad_file_permissions.rs

```diff
@@ -113,13 +115,17 @@ fn py_stat(call_path: &CallPath) -> Option<u16> {
     }
 }
 
-fn int_value(expr: &Expr, semantic: &SemanticModel) -> Option<u16> {
+fn int_value(expr: &Expr, semantic: &SemanticModel) -> Option<Integer> {
     match expr {
         Expr::Constant(ast::ExprConstant {
             value: Constant::Int(value),
             ..
-        }) => value.to_u16(),
-        Expr::Attribute(_) => semantic.resolve_call_path(expr).as_ref().and_then(py_stat),
+        }) => Some(value.clone()),
```

Comment:
Nit: What's the reason for returning an `Integer` here? Converting to a `u16` seems reasonable (because anything larger isn't a valid file permission anyway)

Reply:
the diagnostic would be incorrect if we truncated the number